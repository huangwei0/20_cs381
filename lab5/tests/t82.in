findall(Z, regentWhenBorn('Queen Elizabeth II', Z), Y), 
print(Y), 
print('\n'), 
halt.
