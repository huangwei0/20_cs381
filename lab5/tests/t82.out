[Lady Diana Spencer,Catherine Middleton,Sarah Armstrong-Jones,David: Viscount Linley,Andrew: Duke of York,Edward: Earl of Wessex,Prince William: Duke of Cambridge,Prince Henry of Wales,Sophie Rhys-Jones,Sarah Ferguson,Catherine Middleton,Prince George of Cambridge,Princess Beatrice of York,Princess Eugenie of York,Peter Phillips,Zara Phillips,Autumn Kelly,Mike Tindall,Mia Tindall,Savannah Phillips,Isla Phillips,Lady Louise Windsor,James: Viscount Severn,Serena Armstrong-Jones,Charles Armstrong-Jones,Margarita Armstrong-Jones,Daniel Chatto,Samuel Chatto,Arthur Chatto,Meghan Markle]
